staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -622,3 +531,163 @@ func TestInvalidKeys(t *testing.T) {
 	_, countErr := store.Count(invalidKey)
 	expectInvalidKey("Count", countErr)
 }
+
+func BenchmarkStore_GetList(b *testing.B) {
+	generatePod := func() runtime.Object {
+		return &corev1.Pod{
+			ObjectMeta: metav1.ObjectMeta{Labels: map[string]string{"run": "foo"}},
+			Spec: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{
+						Name:  "foo",
+						Image: "someimage",
+						Ports: []corev1.ContainerPort{
+							{
+								ContainerPort: 80,
+								HostPort:      80,
+							},
+						},
+					},
+				},
+				DNSPolicy:     corev1.DNSClusterFirst,
+				RestartPolicy: corev1.RestartPolicyAlways,
+			},
+			Status: corev1.PodStatus{
+				ContainerStatuses: []corev1.ContainerStatus{
+					{
+						Name: "foo",
+						State: corev1.ContainerState{
+							Running: &corev1.ContainerStateRunning{
+								StartedAt: metav1.Time{},
+							},
+						},
+					},
+				},
+			},
+		}
+	}
+	generateNode := func() runtime.Object {
```

Comment:
I don't think having node here actually provides much value. So let's remove it.

Reply:
fixed